docs/sources/migration-guide/migrating-from-thanos-or-prometheus.md

```diff
@@ -88,6 +88,8 @@ This may cause that incorrect results are returned for the query.
 1) Copy the blocks from Thanos's bucket to an Intermediate bucket
 
    Create an intermediate object storage bucket(S3,GCS,etc.) in your cloud provider where you can copy the historical blocks and work on them before uploading to the Mimir bucket.
+   
+   > **Note**: It is recommeneded to run all the above steps in a **screen** command to avoid any interruptions as these may take time depending on the amount of data to be processed.
```

Comment:
```suggestion
   > **Note**: It is recommeneded to run all of the following steps in a **screen** or **tmux** command to avoid any interruptions as these may take time depending on the amount of data to be processed.
```

Reply:
@pstibrany and @Leegin-darknight, I have updated this suggestion to a tip and tweaked the phrases.

Reply:
@osg-grafana I have updated the PR with the changes suggested. Can you please review?